typeDefinition/ElementChain.d.ts

```diff
@@ -13,8 +13,10 @@ import {
 	BaseChain,
 	UntilModifier,
 } from './modifiers';
+import {DomElementProps} from "./constants/DomElementProps";
 
 export interface ElementChain extends
```

Comment:
I think this will also allow to do `suitest.element().text` which is wrong. Should be probably for ElementEvalChain ? 
@JeKachan

Reply:
@givehug I'am not sure. What should be returned for `await element('myButton').click()` ?

Reply:
`await ...click()` should resolve to `undefined` or maybe `true`.

Reply:
yes ElementQueryChain, my mistake